language/Syntax.md

```diff
@@ -402,7 +402,7 @@ conditional = if 2 > 1 then "ok" else "oops"
 
 ## Let and where bindings
 
-The `let` keyword a collection of local declarations, which may be mutually recursive, and which may include type declarations:
+`let` expressions encompass a list of local declarations, which may be mutually recursive, and which may include type declarations:
```

Comment:
I prefer the previous version with "The `let` keyword introduces a collection of local declarations...", actually.